docker/mongodb.yml

```diff
@@ -8,6 +8,8 @@ services:
       - "127.0.0.1:${MONGO_EXPOSE_PORT:-27017}:27017"
     volumes:
       - mongodb_data:/data/db
+    networks:
+      - webnet
```

Comment:
Don't do that, it is dangerous as it will potentially conflict on prod servers.

Why should you need that ?

Reply:
Otherwise how would other services could access mongodb service if they don't share a common network?
And I'm not worried about the production settings, as it's a dev-only file.

Reply:
When you launch a service, docker-compose creates a default network to which every service is connected. It's named ${COMPOSE_PROJECT_NAME}_default.

But ok, I see that we manually specified `webnet` for now for all services in docker-compose (this is bad, but we shall fix it later).

So ok for that, for now… we will have to fix this network thing in the future.

See https://github.com/openfoodfacts/openfoodfacts-infrastructure/issues/146 (FYI)